collector/mdadm_linux.go

```diff
@@ -190,7 +190,7 @@ func parseMdstat(mdStatusFilePath string) ([]mdStatus, error) {
 
 		// If device is syncing at the moment, get the number of currently synced bytes,
 		// otherwise that number equals the size of the device.
-		if strings.Contains(lines[j], "recovery") || strings.Contains(lines[j], "resync") && !strings.Contains(lines[j], "resync=DELAYED") {
+		if strings.Contains(lines[j], "recovery") || strings.Contains(lines[j], "resync") && !strings.Contains(lines[j], "resync=DELAYED") && !strings.Contains(lines[j], "resync=PENDING") {
```

Comment:
I looked over the linux source[0] and found that these are the only 2 cases.  There is the other style of line that sometimes says `resync = `.  Maybe we can make the parser smarter based on the source.  It also seems like it always uses a `\t` in the code to print out that line.  I can adjust the `Contains`

[0]: http://lxr.free-electrons.com/source/drivers/md/md.c#L7267